test/model.js

```diff
@@ -259,24 +258,20 @@ describe('Model', function() {
     });
     it('should fail athirdroute (user defined route)', function(done) {
       request(app)
-        .put('/api/movies/' + movie1._id + '/athirdroute')
+        .patch('/api/movies/' + movie1._id + '/athirdroute')
         .expect(404, done);
     });
-    it('should allow put of entire object', function(done) {
+    it('should allow PATCH of entire object', function(done) {
       request(app)
         .get('/api/movies/' + config.movies[7]._id)
         .end(function(err, res) {
           var movie = res.body;
           movie.title = 'A different title';
           request(app)
-            .put('/api/movies/' + movie._id)
+            .patch('/api/movies/' + movie._id)
             .send(movie)
             .expect('Content-Type', /json/)
-            .expect(201)
-            .end(function(err, res) {
-              res.body.title.should.equal('A different title');
-              done();
-            });
+            .expect(200, done);
```

Comment:
PATCH should only return a 200, anything else would be an error since you should know the data before and be able to apply the patch client side.

For example if you had this.
```
{
    username: "test",
    email: "old@example.com"
}
```

You'd then patch like this and know the other details.
```
{
    email: "new@example.com"
}
```